pyanaconda/ui/lib/help.py

```diff
@@ -282,6 +282,23 @@ def show_graphical_help(help_path, help_anchor=None):
     else:
         args.append(help_path)
 
-    user = _get_help_user()
-
-    yelp_process = startProgram(["yelp", *args], reset_lang=False, user=user)
+    try:
+        user_name, user_id = _get_help_user()
+        env_prune = ("GDK_BACKEND",)
+        env_add = {
+            "XDG_RUNTIME_DIR": "/run/user/{}".format(user_id),
+            "USER": user_name,
+            "HOME": "/home/{}".format(user_name),
+        }
+    except TypeError:
```

Comment:
It is very hard to see what triggers this exception. If you don't want to check `None` and you prefer to use the exception handler, I suggest to raise an exception from `_get_help_user()`.